charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -382,13 +394,50 @@ def create(self):
                                           name=self.app_name)
                 except CalledProcessError:
                     log('Could not set app name for pool {}'.format(self.name, level=WARNING))
+                nautilus_or_later = cmp_pkgrevno('ceph-common', '14.2.0') >= 0
+                if nautilus_or_later:
+                    # Ensure we set the expected pool ratio
+                    update_pool(client=self.service,
+                                pool=self.name,
+                                settings={'target_size_ratio': str(self.percent_data / 100.0)})
+                if 'pg_autoscaler' in enabled_manager_modules():
+                    try:
+                        enable_pg_autoscale(self.service, self.name)
+                    except CalledProcessError:
+                        log('Could not configure auto scaling for pool {}'.format(self.name, level=WARNING))
             except CalledProcessError:
                 raise
 
     """Get an existing erasure code profile if it already exists.
        Returns json formatted output"""
 
 
+def enabled_manager_modules():
+    """Return a list of enabled manager modules.
+
+    :returns: list[str]
```

Comment:
nit: This should be ``:rtype:`` and list is capitalized.

https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html
https://mypy.readthedocs.io/en/latest/builtin_types.html